shared/javascripts/privly-web/new.js

```diff
@@ -1,266 +0,0 @@
- *
- * These applications can be placed into the following states:
- * 1. Pending Login Check: The app is currently requesting the CSRF
- *    token from the remote server. Callback=pendingLogin
- * 2. Failure to login: The user is not currently authenticated with the
- *    remote server. In this state the user is prompted to login.
- *    Callback=loginFailure
- * 3. Pending post: The user can make the post at this point.
- *    Callback=pendingPost
- * 4. postSubmit: The user submitted the form so the content is being
- *    sent to the remote server. Once it is returned, the URL will
- *    be messaged to the extension (if present) by calling the
- *    "postCompleted" callback.
- * 5. Error creating post: The remote server would not accept the user's
- *    content. The app should display an error message.
- *    Callback=createError
- * 6. Completed post: The remote server has returned a URL. This app should
- *    display it and fire the URL event.
- *    Callback=postCompleted
- */
-var callbacks = {
-  
-  /**
-   * Initialize the whole application.
-   *
-   * @param {function} callback A function to call after the content
-   * returns from the server.
-   */
-  pendingLogin: function(callback) {
-
-    Privly.storage.set("Login:redirect_to_app", window.location.href);
-
-    // Set the nav bar to the proper domain
-    privlyNetworkService.initializeNavigation();
-
-    // Add listeners to show loading animation while making ajax requests
-    $(document).ajaxStart(function() {
-      $('#loadingDiv').show(); 
-    });
-    $(document).ajaxStop(function() { 
-      $('#loadingDiv').hide(); 
-    });
-    
-    // Pass the callback forward
-    function wrapPendingPost() {
-      callbacks.pendingPost(callback);
-    }
-    function wrapLoginFailure() {
-      callbacks.loginFailure(callback);
-    }
-    
-    privlyNetworkService.initPrivlyService(
-      privlyNetworkService.contentServerDomain(), 
-      wrapPendingPost, 
-      wrapLoginFailure, 
-      wrapLoginFailure);
-  },
-  
-  /**
-   * Prompt the user to sign into their server. This assumes the remote
-   * server's sign in endpoint is at "/users/sign_in".
-   *
-   * @param {function} callback A function to call after the UI updates
-   * to tell the user they are not logged in.
-   */
-  loginFailure: function(callback) {
-    
-    privlyNetworkService.showLoggedOutNav();
-    
-    $("#messages").hide();
-    $("#login_message").show();
-    if(callbacks.functionExists(callback)) {
-      callback();
-    }
-  },
-  
-  /**
-   * Tell the user they can create their post by updating the UI.
-   *
-   * @param {function} callback A function to call after the login check
-   * was successfull.
-   */
-  pendingPost: function(callback) {
-    
-    privlyNetworkService.showLoggedInNav();
-
-    // Listen for a message containing the initial content for the form
-    // TODO: no receivers?
-    Privly.message.messageExtension({'ask': 'initialContent'}, true)
```

Comment:
Those codes are removed because responses are not sent anywhere after refaction in privly-chrome.

To add it back, you need to:
1) request for initial content at: https://github.com/privly/privly-applications/pull/274/files#diff-045fb9b2b77dc665412fe50ccef3b6fdR281
2) response for initial content requesting somewhere in the content script.

do you have ideas of point 2?